tests/perf/report/charts/v1.0/service_invocation/RESULTS.md

```diff
@@ -0,0 +1,83 @@
+# Service Invocation Performance
```

Comment:
Im open to calling this a diff filename. This is a direct copy pasta from the [docs here](https://docs.dapr.io/operations/performance-and-scalability/perf-service-invocation/), converted to markdown. The idea is to make the source of truth repeatable using our perf test suite here in dapr/dapr.